sdk/eventhub/Microsoft.Azure.Management.EventHub/tests/Tests/ScenarioTests.NamespaceBYOKTests.CRUD.cs

```diff
@@ -13,13 +13,14 @@ namespace EventHub.Tests.ScenarioTests
     using Microsoft.Azure.Management.EventHub.Models;
     using Microsoft.Azure.Management.KeyVault;
     using Microsoft.Azure.Management.KeyVault.Models;
+    using Microsoft.Azure.Management.ManagedServiceIdentity;
     using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
     using TestHelper;
     using Xunit;
     public partial class ScenarioTests
     {
         [Fact]
-        public void NamespaceBYOKCreateGetUpdateDelete()
+        public void NamespaceBYOKCreateGetUpdateDelete1()
```

Comment:
It looks strange that we add `1` in the end of this method name, any reason for this?